runtime/doc/builtin.txt

```diff
@@ -4261,6 +4262,25 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2})				*getregion()*
+		Returns the list of strings from {pos1} to {pos2}.
+		For possible values of {pos1} and {pos2} see |line()|.
+		This function uses the line and column number from the
+		specified position.
+		It is useful to get text starting and ending in different
+		columns, such as |characterwise-visual| selection.
+
+		Note that:
+		- {pos1} and {pos2} are swapped if {pos1} is after {pos2}.
+		- If |virtualmode| is enabled and selection is past the end of
```

Comment:
I think you meant "virtualedit" here instead of "virtualmode".

Reply:
Thanks.  I will fix it.